### PR TITLE
Add PRO.Narrow for ejb home lookup

### PR DESCRIPTION
**Pull request by olendvcook**
#16238

### CODE REVIEW

**Comment by olendvcook** [comment]
#build

**Comment by olendvcook** [comment]
#build

File: dev/com.ibm.ws.ejbcontainer.remote_fat/fat/src/com/ibm/ws/ejbcontainer/remote/fat/tests/RemoteTests.java [hunk 1]
```diff
@@ -155,6 +155,16 @@ public static void beforeClass() throws Exception {
 
         ShrinkHelper.exportDropinAppToServer(server, CrossAppRemoteEJB, DeployOptions.SERVER_ONLY);
 
+        //#################### CrossApp2xEJB.war
```
Comment (tkburroughs) [inline dev/com.ibm.ws.ejbcontainer.remote_fat/fat/src/com/ibm/ws/ejbcontainer/remote/fat/tests/RemoteTests.java:158]: Comment is incorrect, should be "CrossApp2xTest.ear".

**Review by tkburroughs — APPROVED** [review]
Only one minor comment issue in a test; probably not important to fix; approving.

**Comment by olendvcook** [comment]
#run-libby-bot

**Pull request merged**